src/process/ProcessManager.h

```diff
@@ -49,11 +49,12 @@ class ProcessManager : public std::enable_shared_from_this<ProcessManager>,
 {
   public:
     static std::shared_ptr<ProcessManager> create(Application& app);
-    virtual ProcessExitEvent runProcess(std::string const& cmdLine,
-                                        std::string outputFile = "") = 0;
+    virtual std::shared_ptr<ProcessExitEvent>
```

Comment:
if we move to using to `std::shared_ptr<ProcessExitEvent>` across the board, should we make `ProcessExitEvent` non-copyable/movable?